### DISCUSSION TITLE
Does Qdrant support data encryption?

### DISCUSSION THREAD

**Discussion by hanqiushi** [qdrant/qdrant#3188] [General]
I have a question.
Maybe we want to use qdrant locally, and don't want other people to manipulate with data inserted.
Does qdrant support data encryption? Or will support this kind of feature later?
Thanks

**Comment by generall** [comment]
Qdrant supports API encryption using TLS, including inter-node communication.
For access control we allow API keys.
Storage encryption is usually solved by external tool, which would encrypt the whole volume.

Do you have something else in mind?

**Reply by kiradeva** [reply]
What about Qdrant cloud?  How is storage encryption handled there?
